[lldb/crashlog] Always load Application Specific Backtrace Thread images

medismailben opened the pull request:
This patch changes the crashlog image loading default behaviour to not only load images from the crashed thread but also for the application specific backtrace thread.

This patch also move the Application Specific Backtrace / Last Exception Backtrace tag from the thread queue field to the thread name.

rdar://128276576

JDevlieghere reviewed (approved):
LGTM. Can we test the new behavior?

medismailben committed:
[lldb/crashlog] Always load Application Specific Backtrace Thread images

This patch changes the crashlog image loading default behaviour to not
only load images from the crashed thread but also for the application
specific backtrace thread.

This patch also move the Application Specific Backtrace / Last Exception
Backtrace tag from the thread queue field to the thread name.

rdar://128276576

Signed-off-by: Med Ismail Bennani <ismail@bennani.ma>
